.babelrc.js

```diff
@@ -16,5 +16,8 @@ module.exports = {
   ],
   plugins: [
     '@babel/plugin-proposal-function-bind',
+    // `loose:true` produces `foo==null` and it's fine because we'll never use `document.all`
+    // which is the only reason for the default `false`
+    ['@babel/plugin-proposal-optional-chaining', { loose: true }],
```

Comment:
This is already supported in `@gera2ld/plaid/config/babelrc`.

Reply:
Great!